org.jacoco.doc/docroot/doc/maven.html

```diff
@@ -91,7 +91,7 @@ <h2>Usage</h2>
 &lt;plugin&gt;
   &lt;groupId&gt;org.jacoco&lt;/groupId&gt;
   &lt;artifactId&gt;jacoco-maven-plugin&lt;/artifactId&gt;
-  &lt;version&gt;@project.version@&lt;/version&gt;
+  &lt;version&gt;@{project.version}&lt;/version&gt;
```

Comment:
Should read: `${project.version}` (dollar)

Reply:
Ouch. Fixed!